src/streamlink/plugins/wwenetwork.py

```diff
@@ -21,7 +20,7 @@
 
 
 @pluginmatcher(re.compile(
-    r"https?://watch\.wwe\.com/(channel)?",
+    r"https?://network\.wwe\.com/video/(?P<video_id>\d+)?",
```

Comment:
Are these the only stream URLs on that site?
Are these live streams or just vods?

If it's just VODs, then the plugin doesn't meet the required plugin criteria and should be removed instead, since we don't keep plugins which are just for VOD content.
https://github.com/streamlink/streamlink/blob/master/CONTRIBUTING.md#plugin-requests

Reply:
Sorry forgot about the live streams, yes there is an option to take them, I changed the code, now

if https://network.wwe.com/ - we get live stream
if https://network.wwe.com/videщ/<video_id> - we get VOD with id = video_id